contrib/wireshark/template.py

```diff
@@ -0,0 +1,82 @@
+pkgname = "wireshark"
+pkgver = "4.2.2"
+pkgrel = 0
+build_style = "cmake"
+configure_args = [
+    "-DENABLE_GNUTLS=ON",
+    "-DENABLE_LUA=ON",
+    "-DENABLE_DEBUG=OFF",
+    "-DUSE_qt6=ON",
+]
+make_build_args = ["--target", "all", "test-programs"]
+hostmakedepends = [
+    "asciidoctor",
+    "cmake",
+    "doxygen",
+    "flex",
+    "gettext-devel",
+    "libcap-progs",
+    "ninja",
+    "pkgconf",
+    "qt6-qtbase",
+    "qt6-qttools",
+]
+makedepends = [
+    "c-ares-devel",
+    "glib-devel",
+    "gnutls-devel",
+    "heimdal-devel",
+    "libcap-devel",
+    "libgcrypt-devel",
+    "libnl-devel",
+    "libpcap-devel",
+    "libssh-devel",
+    "libxml2-devel",
+    "lua5.1-devel",
+    "lz4-devel",
+    "minizip-devel",
+    "nghttp2-devel",
+    "opus-devel",
+    "pcre2-devel",
+    "perl",
+    "portaudio-devel",
+    "qt6-qt5compat-devel",
+    "qt6-qtbase-devel",
+    "qt6-qtmultimedia-devel",
+    "qt6-qtsvg-devel",
+    "qt6-qttools-devel",
+    "snappy-devel",
+    "spandsp-devel",
+    "speexdsp-devel",
+    "zlib-devel",
+    "zstd-devel",
+]
+checkdepends = ["python-pytest-xdist"]
+pkgdesc = "Network protocol analyzer"
+maintainer = "psykose <alice@ayaya.dev>"
+license = "GPL-2.0-or-later"
+url = "https://www.wireshark.org"
+source = f"https://www.wireshark.org/download/src/all-versions/wireshark-{pkgver}.tar.xz"
+sha256 = "9e3672be8c6caf9279a5a13582d6711ab699ae2a79323e92a99409c1ead98521"
+# forbid non-wireshark-group users from reading all network packets
+file_modes = {
+    "usr/bin/dumpcap": ("root", "_wireshark", 0o750),
+}
```

Comment:
this actually doesn't work and the file is installed as 755. maybe it breaks because of file_xattrs at the same time?

Reply:
hm no, even with no xattrs it doesn't work

but it does work for dbus so i'm confused

Reply:
this is the executable perms sanitization code, it skips it for suid but does not account for xattrs (i'll fix that later)

Reply:
should be addressed properly by https://github.com/chimera-linux/cports/commit/37ef5a5f5aeac6592138a7eef05d7043b6611cae